proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -458,8 +458,11 @@ message ActivityInfo {
         int64 last_redirect_counter = 2;
     }
 
-    // fist time activity was schedulled.
+    // First time the activity was scheduled.
     google.protobuf.Timestamp first_scheduled_time = 39;
+
+    // Last time an activity failure was recorded by the server.
+    google.protobuf.Timestamp last_attempt_complete_time = 40;
```

Comment:
Want to make sure this is replicated. Can you add an XDC test to verify? Can do that in a followup PR too if you prefer.

Reply:
If this is needed - I would prefer to do that in a separate PR.